iotdb-core/metrics/core/src/main/java/org/apache/iotdb/metrics/core/type/IoTDBCounter.java

```diff
@@ -17,29 +17,35 @@
  * under the License.
  */
 
-package org.apache.iotdb.metrics.micrometer.type;
+package org.apache.iotdb.metrics.core.type;
 
 import org.apache.iotdb.metrics.type.Counter;
 
-public class MicrometerCounter implements Counter {
+import java.util.concurrent.atomic.LongAdder;
+
+public class IoTDBCounter implements Counter {
+  private final LongAdder count;
+  // Here we retain the micrometer counter variable in order to use its naming system within
+  // micrometer.
   io.micrometer.core.instrument.Counter counter;
```

Comment:
If we just want to use its naming system, do we need to hold a reference here? Seems that it is never used?